### PR TITLE
Changes in CI and build.sbt

### PR DESCRIPTION
**Pull request by Pask423**

### CODE REVIEW

**Comment by adamw** [comment]
Great! I wanted to do that for a long time. This follows how things are done in tapir, right?

One thing missing is mergify configuration so that merging steward PR works

**Comment by Pask423** [comment]
Yes, exactly. Sorry I forgot about .mergify config.

File: build.sbt [hunk 1]
```diff
@@ -232,26 +240,56 @@ lazy val allAggregates = projectsWithOptionalNative ++
...
+
 lazy val rootProject = (project in file("."))
   .settings(commonSettings: _*)
   .settings(
     publish / skip := true,
     name := "sttp",
     testJVM := (Test / test).all(filterProject(p => !p.contains("JS") && !p.contains("Native"))).value,
-    testJS1 := (Test / test)
-      .all(filterProject(p => p.contains("JS") && (p.contains("2_11") || p.contains("2_12"))))
+    testJS2_11 := (Test / test)
```
Comment (adamw) [inline build.sbt:272]: how does this differ from `testScoped`?

Comment (Pask423) [reply build.sbt:272]: JS tests are run in different way then others there is a comment in ci file, it works the same way in tapir

Comment (adamw) [reply build.sbt:272]: Hm which comment?

**Pull request merged**